eos-fix-home-dir-permissions.service

```diff
@@ -0,0 +1,28 @@
+# Clean up any home directories which were created with
+# default permissions 755 rather than 700
+
+[Unit]
+Description=Fix existing home directory permissions from 755 to 700
+DefaultDependencies=no
+Conflicts=shutdown.target
+Wants=local-fs.target
+After=local-fs.target
+
+# Only run on updates
+Before=multi-user.target systemd-update-done.service
+ConditionNeedsUpdate=/etc
+
+# Only run if the home dir permissions have not yet been cleaned up.
+# The default dir mode was changed for 3.1.6,
+# and we only want to perform this cleanup once, in case
+# an admin later intentionally changes home dir permissions.
+ConditionPathExists=!/var/eos-dir-mode-700
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+StandardOutput=journal+console
```

Comment:
I don't think this setting is needed. It's added in some of the other services here so that you get logging on the console. That's because some of the other services run early and have real effects on boot. If something goes wrong, then maybe you can see something on the console.

This service runs much later when the journal will be normally accessible and messages can be gathered from there.